tests/Model/ZoneTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace SilverCommerce\GeoZones\Tests;
+
+use Generator;
+use Locale;
+use SilverCommerce\GeoZones\Model\Zone;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\i18n\i18n;
+
+class ZoneTest extends SapphireTest
+{
+    private const LOCALE = "en_GB";
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        i18n::set_locale(self::LOCALE);
+    }
+
+    public function testDefaultsToCurrentLocale()
+    {
+        
+        $zone = Zone::create();
+        $this->assertEquals(self::LOCALE, $zone->Country);
```

Comment:
It looks like there is a bug here, should the Country not just be GB, the `populateDefaults` part of this object picks up on the default (or set) Locale but the `Country` is never separated. See comment in `Zone::class`